test-cases/longevity/longevity-user-profile-d.yaml

```diff
@@ -0,0 +1,32 @@
+test_duration: 360
+
+service_level_shares: [2000]
```

Comment:
Maximum allowed service level value is 1000. Do you see that service_level with `2000` shares is created? If yes - it looks like an issue

Reply:
In this job , it is mostly used as role name, because on master there is no SLA. 
But i will change it to 1000, so it also could run on enterprise